plugins/Monolog/Formatter/LineMessageFormatter.php

```diff
@@ -23,17 +23,22 @@ class LineMessageFormatter implements FormatterInterface
      * @var string
      */
     private $logMessageFormat;
-
+    private $excludePatterns;
+    private $customFunction;
     private $allowInlineLineBreaks;
 
     /**
      * @param string $logMessageFormat
      * @param bool $allowInlineLineBreaks If disabled, a log message will be created for each line
      */
-    public function __construct($logMessageFormat, $allowInlineLineBreaks = true)
+    public function __construct($logMessageFormat, $allowInlineLineBreaks = true, $excludePatterns = null, $customFunctionFile = null)
     {
         $this->logMessageFormat = $logMessageFormat;
         $this->allowInlineLineBreaks = $allowInlineLineBreaks;
+        $this->excludePatterns = $excludePatterns;
+        if (gettype($customFunctionFile) === "string") {
```

Comment:
you can add typehint to the constructor param and don't need to check for it being a string, just not empty.